scripts/cypress/plugins/index.js

```diff
@@ -12,10 +12,74 @@
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
 
+// @ts-check
+const path = require('path');
+const fs = require('fs');
+const { startDevServer } = require('@cypress/webpack-dev-server');
+const { TsconfigPathsPlugin } = require('tsconfig-paths-webpack-plugin');
+
 /**
+ * Cypress Webpack devServer that uses esbuild-loader for speed,
  * @type {Cypress.PluginConfig}
  */
 module.exports = (on, config) => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
+  const tsConfigBasePath = path.resolve(__dirname, '../../../tsconfig.base.json');
```

Comment:
Does this also work for v8?

Reply:
yes, in the sense that if there are no aliases, it tries to resolve the modules in the standard way